lib/widget/training_information/post_op_home_training.dart

```diff
@@ -0,0 +1,317 @@
+import 'post-op-hos-day1/nutrition_advice.dart';
+import 'post-op-hos-day1/respiratory_advice.dart';
+import 'post-op-hos-day2-7/behave_advice.dart';
+import 'post-op-hos-day2-7/digestive_rehabilitation_advice.dart';
+import 'post-op-hos-day2-7/drain_secretion_advice.dart';
+import 'post-op-hos-day2-7/pain_advice_day2.dart';
+import 'post-op-hos-day2-7/pulmanary_rehabilitation_advice.dart';
+
+class PostOpHomeTrainingPage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _PostOpHomeTrainingPageState();
+}
+
+class _PostOpHomeTrainingPageState extends State<PostOpHomeTrainingPage> {
+  int number;
+  String topic;
+  String state;
+  void detailpage(int number) {
+    if (number == 1) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => RespiratoryAdviceDay0()));
+    } else if (number == 2 || number == 5) {
+      Navigator.push(
+          context, MaterialPageRoute(builder: (context) => PainAdviceDay0()));
+    } else if (number == 3 || number == 6) {
+      Navigator.push(
+          context, MaterialPageRoute(builder: (context) => DrainAdviceDay0()));
+    } else if (number == 4) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => RespiratoryAdviceDay1()));
+    } else if (number == 7) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => BloodclotsAdviceDay1()));
+    } else if (number == 8) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => NutritionAdviceDay1()));
+    } else if (number == 9) {
+      Navigator.push(
+          context, MaterialPageRoute(builder: (context) => PainAdviceDay2()));
+    } else if (number == 11) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => DrainSecretionAdviceDay2()));
+    } else if (number == 12) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => PulmonaryAdviceDay2()));
+    } else if (number == 13) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => DigestiveAdviceDay2()));
+    } else if (number == 14) {
+      Navigator.push(
+          context, MaterialPageRoute(builder: (context) => BehaveAdviceDay2()));
+    } else if (number == 15) {
+      Navigator.push(
+          context, MaterialPageRoute(builder: (context) => PainAdvice()));
+    } else if (number == 16 || number == 10) {
+      Navigator.push(
+          context, MaterialPageRoute(builder: (context) => InfectionAdvice()));
+    } else if (number == 17) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => SurgicalIncisionAdvice()));
+    } else if (number == 18) {
+      Navigator.push(context,
+          MaterialPageRoute(builder: (context) => DailyActivityAdvice()));
+    } else if (number == 19) {
+      Navigator.push(
+          context, MaterialPageRoute(builder: (context) => FoodAdvice()));
+    }
+  }
+
```

Comment:
Should use ENUM instead of plain integer.

Reply:
I will try ka 

Thank you